kerastuner/applications/efficientnet.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Hypertunable version of EfficientNet based on Keras.applications."""
+
+import tensorflow as tf
+import tensorflow.keras as keras
+
+from tensorflow.keras import layers
+from tensorflow.keras.applications import efficientnet
+from tensorflow.keras.layers.experimental import preprocessing
+
+from kerastuner.engine import hypermodel
+
+
+EFFICIENTNET_MODELS = {'B0': efficientnet.EfficientNetB0,
+                       'B1': efficientnet.EfficientNetB1,
+                       'B2': efficientnet.EfficientNetB2,
+                       'B3': efficientnet.EfficientNetB3,
+                       'B4': efficientnet.EfficientNetB4,
+                       'B5': efficientnet.EfficientNetB5,
+                       'B6': efficientnet.EfficientNetB6,
+                       'B7': efficientnet.EfficientNetB7}
+
+EFFICIENTNET_IMG_SIZE = {'B0': 224,
+                         'B1': 240,
+                         'B2': 260,
+                         'B3': 300,
+                         'B4': 380,
+                         'B5': 456,
+                         'B6': 528,
+                         'B7': 600}
+
+
+class HyperEfficientNet(hypermodel.HyperModel):
+    """An EfficientNet HyperModel.
```

Comment:
We should specify in the docstring what format we expect the data to be in (i.e is it ints [0, 255] or floats [0, 1], etc)

Reply:
Thanks for the comment. The docstring is updated.